src/query/config/src/inner.rs

```diff
@@ -29,6 +29,7 @@ use common_tracing::Config as LogConfig;
 use common_users::idm_config::IDMConfig;
 
 use super::outer_v0::Config as OuterV0Config;
+use super::outer_v0::MetaType;
```

Comment:
We should not depend on any types added in `outer_v0`. `MetaType` should be declared in `inner`.